Assets/Scripts/UI/Components/Specific_Components/NL3D_Openen_Opslaan_UI.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Netherlands3D.Twin
+{
+    public class NL3D_Openen_Opslaan_UI : MonoBehaviour
+    {
+        public Slider Loadingbar;
```

Comment:
Stijlverbetering voor het vervolg: kan je hier `[SerializeField] private` van maken ipv public? Alleen als het echt nodig is om vanaf een ander component hieraan te refereren willen we het public maar als het alleen voor de editor is dan heeft private de voorkeur. Dit helpt om te voorkomen dat vanuit andere classes deze gewijzigd wordt en daarmee mogelijk bugs komen die we niet hadden verwacht :)